pandas/tests/io/formats/test_to_html.py

```diff
@@ -1905,6 +1905,636 @@ def test_to_html_multiindex_max_cols(self):
         </table>""")
         assert result == expected
 
+    def test_to_html_multi_indices_index_false(self):
```

Comment:
Haven't looked in detail at all of these if only due to the sheer volume of them, but is there not a way for you to parametrize these instead?